src/crud.js

```diff
@@ -0,0 +1,64 @@
+export const tasks = JSON.parse(localStorage.getItem('tasksList')) || [];
+
+function addTaskToStorage() {
+  const tasksInfo = JSON.stringify(tasks);
+  localStorage.setItem('tasksList', tasksInfo);
+}
+
+export function AddTask() {
+  const task = {
+    description: document.querySelector('.input-task').value,
+    completed: false,
+    index: tasks.length,
+  };
+
+  tasks.push(task);
+  addTaskToStorage();
+}
+
+export function deleteCompletedTask(taskArr) {
+  const storeLength = JSON.parse(localStorage.getItem('tasksList')).length;
+
+  taskArr = taskArr.filter((task) => task.completed === false);
+
+  localStorage.setItem('tasksList', JSON.stringify(taskArr));
+
+  window.location.reload();
+  taskArr.forEach((task, i) => task.index === Array.from(Array(storeLength).keys())[i]);
+  localStorage.setItem('tasksList', JSON.stringify(taskArr));
+}
+
+function moveToTrash(taskArr) {
+  const trashCan = [...document.querySelectorAll('.trash')];
+
+  trashCan.forEach((can) => can.addEventListener('click', () => {
+    const storeLength = JSON.parse(localStorage.getItem('tasksList')).length;
+    taskArr = taskArr.filter((task) => task.index !== parseInt(can.id[6], 10));
+    localStorage.setItem('tasksList', JSON.stringify(taskArr));
+    window.location.reload();
+    taskArr.forEach((taskArr, i) => taskArr.index === Array.from(Array(storeLength).keys())[i]);
+    localStorage.setItem('tasksList', JSON.stringify(taskArr));
+  }));
+}
```

Comment:
- [ ] You have to update the index of the **tasks** when it's removed.

  ![lslsl](https://user-images.githubusercontent.com/61562626/134667402-49db84ce-9c9e-4682-a761-8553633e3fcf.jpeg)


![mm](https://user-images.githubusercontent.com/61562626/134667379-efaff6eb-ab55-400d-b492-76676fe9140c.png)